js/templates/modals/settings/advanced/advanced.html

```diff
@@ -17,25 +17,27 @@ <h2 class="h3 clrT"><%= ob.polyT('settings.advancedTab.sectionName') %></h2>
                 <div class="clrT2 txSm"><%= ob.polyT('settings.advancedTab.helperVisualEffects') %></div>
               </div>
               <div class="col9">
-                <div class="btnRadio clrBr">
-                  <input type="radio"
-                         data-persistence-location="local"
-                         name="showAdvancedVisualEffects"
-                         value="true"
-                         id="advancedTabVisualEffectsOn"
-                         data-var-type="boolean"
-                  <% if(ob.showAdvancedVisualEffects) { %>checked<% } %>>
-                  <label for="advancedTabVisualEffectsOn"><%= ob.polyT('settings.on') %></label>
-                </div>
-                <div class="btnRadio clrBr">
-                  <input type="radio"
-                         data-persistence-location="local"
-                         name="showAdvancedVisualEffects"
-                         value="false"
-                         id="advancedTabVisualEffectsOff"
-                         data-var-type="boolean"
-                  <% if(!ob.showAdvancedVisualEffects) { %>checked<% } %>>
-                  <label for="advancedTabVisualEffectsOff"><%= ob.polyT('settings.off') %></label>
+                <div class="btnStrip">
```

Comment:
The buttons here were put in a button strip container, radio buttons won't line up horizontally on their own any more since that was causing some issues.